app/models/service_provider_request.rb

```diff
@@ -12,7 +12,8 @@ def initialize(
     ial: nil,
     aal: nil,
     requested_attributes: [],
-    biometric_comparison_required: false
+    biometric_comparison_required: false,
+    vtr: nil # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
should we add `acr` too? so we dont have to persisnt ial//aal separately?
```suggestion
    vtr: nil, # rubocop:disable Lint/UnusedMethodArgument
    acr: nil # rubocop:disable Lint/UnusedMethodArgument
```

Reply:
I have wavered on whether we should add ACR separately. They get split out and parsed separately by both the OIDC controller and the SAML request.

I was imagining something like this (at least for now):

```ruby
class ServiceProviderRequest
  def acr_values
    [ial, aal].join(' ')
  end
end
```

Eventually we may get to a place where we can pass in a raw ACR value. I'm not sure whether that is better to go for now or hold off on. I haven't spent enough time hacking on either the SAML or OIDC portion to know which is better.

Reply:
Okay, I did some poking around and decided to go ahead and throw the raw "acr_values" prop on there